src/com/android/deskclock/AlarmClockFragment.java

```diff
@@ -1502,10 +1502,12 @@ private String getRingToneTitle(Uri uri) {
             if (title == null) {
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit